Web_Workflow_Quickstart/env.py

```diff
@@ -6,79 +6,87 @@
 import storage
 import microcontroller
 
-# Get all files in the format of .env.xxxxxxxxxx
-def enumerate_env_files():
+SETTINGS_FOLDER = "/"
+
+# Get all files in the format of xxxxxxxxxx.toml except settings.toml
+def enumerate_toml_files():
     found_files = []
-    all_files = os.listdir("/")
+    all_files = os.listdir(SETTINGS_FOLDER)
     for current_file in all_files:
-        if current_file[:4] == ".env" and len(current_file) > 4:
-            found_files.append(current_file)
+        if (
+            not current_file.startswith("._")
+            and not current_file.endswith(".toml")
```

Comment:
Whoops, I introduced a bug! 
```suggestion
            and current_file.endswith(".toml")
```